README.md

```diff
@@ -79,6 +79,32 @@ Here's an example with a service provider list:
   }
 ```
 
+## Automatic Modem
+
+The `VintageNetMobile.Modem.Automatic` implementation is a modem implementation
+that will try to discover the attached modem. This is useful if you need to
+support a range of modems and cannot provide a static configuration for one
+particular modem.
```

Comment:
```suggestion
If it is not known ahead of time what modem will be installed, the `VintageNetMobile.Modem.Automatic` implementation can detect and configure it for you. It works by waiting for the system to report a USB device
with a known vendor and product ID. Once one is found, it calls `VintageNet.configure/3` to change the configuration from using the `Automatic` modem to the actual modem implementation. It tells `VintageNet.configure/3` to not persist the configuration so that the detection process runs again on the next boot. 

Here's an example:
```